modules/shared/partials/acid-transactions.adoc

```diff
@@ -419,6 +394,7 @@ Currently, Distributed ACID Transactions are available for:
 * The xref:2.4@go-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[Go SDK].
 * The xref:3.3@java-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[Java SDK].
 * The xref:4.0@nodejs-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[node.js SDK].
+<<<<<<< Updated upstream
```

Comment:
Intended?

Reply:
Hah, nope, let me double check why I got that in there and clean this up.